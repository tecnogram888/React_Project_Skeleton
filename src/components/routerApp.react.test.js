import React from 'react';
import { Provider } from 'react-redux'
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store'
import RouterApp from './router.js';

test('Snapshot Test Practice -- Outline of whole app renders', () => {
    const store = {
        text: 'hello world',
    }
    const mockStore = configureStore()

    const component = renderer.create(
        <Provider store={mockStore(store)}>
            <RouterApp />
        </Provider>)
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
