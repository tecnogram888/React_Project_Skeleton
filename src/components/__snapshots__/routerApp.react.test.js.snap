// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test Practice -- Outline of whole app renders 1`] = `
<div>
  <div>
    <h1
      className="asdf"
    >
      Welcome
    </h1>
    <p>
      <a
        href="/login/"
        onClick={[Function]}
      >
        Login
      </a>
    </p>
    <p>
      <a
        href="/app1/"
        onClick={[Function]}
      >
        App1
      </a>
    </p>
    <p>
      <a
        href="/app2/"
        onClick={[Function]}
      >
        App2
      </a>
    </p>
  </div>
</div>
`;
